action.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Release Pull Request & Changelog Creator'
+description: 'Release Pull Request & Changelog Creator'
+author: 'Compucorp'
+
+inputs:
+  GITHUB_TOKEN:
+    description: 'Github Token'
+    required: true
+  VERSION_NAME:
+    description: 'Release Version Name'
+    required: true
+  REPLACE:
+    description: 'Replace Commands'
+    required: true
+  FILE_UPDATE_COMMIT_DESC:
+    description: 'Commit name to update the file'
+    required: true
+  RELEASE_PR_TITLE:
+    description: 'Pull Request Title'
+    required: true
+  RELEASE_PR_IDENTIFIER_LABEL:
+    description: 'Label to Identify Release Pull Request'
+    required: true
+
+runs:
+  using: 'node12'
+  main: 'dist/index.js'
```

Comment:
No this is expected for github actions. Either we have to push node-modules folder, or build a single file. Because it is going to be directly used by Github, and we cant have any intermediate steps to build the file.
https://docs.github.com/en/actions/creating-actions/creating-a-javascript-action#commit-tag-and-push-your-action-to-github